classes/notification/managerDelegate/ConfigurationNotificationManager.inc.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file classes/notification/managerDelegate/ConfigurationNotificationManager.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ConfigurationNotificationManager
+ * @ingroup managerDelegate
+ *
+ * @brief Configuration notification types manager delegate.
+ */
+
+use PKP\notification\NotificationManagerDelegate;
+use PKP\db\DAORegistry;
+
+class ConfigurationNotificationManager extends NotificationManagerDelegate {
+
+	/**
+	 * Constructor.
+	 * @param $notificationType int NOTIFICATION_TYPE_...
+	 */
+	function __construct($notificationType) {
+		parent::__construct($notificationType);
+	}
+
+	/**
+	 * @copydoc PKPNotificationOperationManager::getNotificationMessage()
+	 */
+	public function getNotificationMessage($request, $notification) {
+		$notificationSettingsDao = DAORegistry::getDAO('NotificationSettingsDAO');
+		$notificationSettings = $notificationSettingsDao->getNotificationSettings($notification->getId());
+		assert(!is_null($notificationSettings['contents']));
+		switch ($notification->getType()) {
+			case PKPNotification::NOTIFICATION_TYPE_CONFIGURE_USER:
+				return $notificationSettings['contents'];
+			case PKPNotification::NOTIFICATION_TYPE_CONFIGURE_PLUGIN:
+				return $notificationSettings['contents'];
+			default:
+				assert(false);
```

Comment:
I know you're following existing patterns, but in new code, we're moving more to using `default: throw new \Exception(...)` in place of the old `default: assert(false);` patterns. (Here and below.)